numba/cuda/decorators.py

```diff
@@ -104,6 +105,8 @@ def device_jit(func):
                                   debug=debug)
 
         if device:
+            msg = ("Eager compilation of device functions is deprecated")
```

Comment:
Should this message also remind users that the eagerness is due to signature(s) being provided?

Reply:
Thanks for the suggestion - it does now.